src/libdrakvuf/vmi.c

```diff
@@ -446,6 +459,16 @@ event_response_t pre_mem_cb(vmi_instance_t vmi, vmi_event_t* event)
         pass->proc_data.pid       = proc_data.pid;
         pass->proc_data.ppid      = proc_data.ppid;
         pass->proc_data.userid    = proc_data.userid;
+        pass->proc_data.tid       = proc_data.tid;
+        pass->attached_proc_data.base_addr = trap_info.attached_proc_data.base_addr;
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wcast-qual"
```

Comment:
Could we  get this working without having to disable the warning?

Reply:
Yes I can.

Without this I get the error:
```
vmi.c:327:27: error: passing argument 1 of ‘g_free’ discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]
     g_free(pass->proc_data.name);
```

But it I remove `const` qualifier from `struct process_data` field `name` (see d9fa5e2) than everything is OK.

Reply:
Have a local non-const version of variable, you can pass that to g_free.

Reply:
@tklengyel I have fixed this in d9fa5e2. You don't like this commit?

Reply:
Sorry I don't, the proc_data char* field needs to remain const.

Reply:
That's all right. I will fix it.